.changeset/witty-wombats-provide.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/core-components': minor
+'@backstage/theme': minor
+---
+
+Add configurable gradient property to backstage theme
```

Comment:
```suggestion
Add a configurable `palette.bursts.gradient` property to the Backstage theme, to support customizing the gradients in the `ItemCard` header.
```

Reply:
Updated the description to contain more detail as well.